### PR TITLE
fixed buttons size

### LINKED ISSUE #391: Button style standardization
1. Buttons should not have a min-width, and the current min-height of 60px (style.css:242) is too tall (should be more like 50?) 
2. Need a black button style
3. Offer-details section style buttons:
This section style puts the linked text under the button. 
- I think the space between the button and text can be smaller (their blog button spacing was larger, but let's standardaize).  
- Some buttons have the text left-aligned, some centered. Let's standardize and center it under the button.
- until now we only saw the text against dark backgrounds, so the text is white. We need another option for the regular link color.

**I think we should use this as the best standardized button example, with regard to position/size/spacing:**
image (seen in the Marquee banner on their home page).

Their buttons, in the tabs on the home page:
image

Ours, non-blog pages:
image
image

Theirs on blog pages: (ignore the square shape)
image

Ours on blog pages:  (they agreed to use round style)
image

https://main--sling--aemsites.aem.page/

The fragment google doc is here
https://www.sling.com/aemedge/fragments/get-amc
https://docs.google.com/document/d/1rMqzNZvXUoac8-pQE3klmbuj0KD_UVGFtnZsugskNKg/edit?tab=t.0

### PR DESCRIPTION
**Pull request by rusmeenkhan1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #391 

Test URLs:
- Before: https://main--sling--aemsites.aem.live/
- After: https://391-buttons--sling--aemsites.aem.page/

- Before: https://main--sling--aemsites.aem.live/aemedge/fragments/get-amc
- After: https://391-buttons--sling--aemsites.aem.page/aemedge/fragments/get-amc

### CODE REVIEW

**Comment by rusmeenkhan1** [comment]
Still WIP.

**Review by helms-charity — CHANGES_REQUESTED** [review]
The buttons shouldn't have a min-width I don't think. Compare to their site, https://www.sling.com/#freestream 
And they still have too much padding around the text.

**Comment by helms-charity** [comment]
Ouch I realized where we are already using bold.  Links that are within other text should NOT be given the button-container class.
image

**Comment by rusmeenkhan1** [comment]
Yes Charity, i also noticed this.

**Review by helms-charity — COMMENTED** [review]
I'm not seeing a black button for AMC+ on the page linked in this ticket, https://391-buttons--sling--aemsites.aem.page/

**Review by helms-charity — APPROVED** [review]
I think this looks good now

File: aemedge/styles/styles.css [hunk 1]
```diff
@@ -233,13 +241,12 @@ a.button::-moz-focus-inner {
 a.button.primary,
 button.primary,
 a.button.secondary,
-button.secondary {
+button.secondary,a.button.dark {
   border-radius: 1600px;
   opacity: 1;
-  min-width: 11.25rem;
   font-size: 1rem;
```
Comment (helms-charity) [inline aemedge/styles/styles.css:247]: Live site has font size of 18px for buttons; use this REM size (or if we have a variable for it
`font-size: 1.125rem;`

File: aemedge/styles/styles.css [hunk 2]
```diff
@@ -233,13 +241,12 @@ a.button::-moz-focus-inner {
 a.button.primary,
 button.primary,
 a.button.secondary,
-button.secondary {
+button.secondary,a.button.dark {
   border-radius: 1600px;
   opacity: 1;
-  min-width: 11.25rem;
-  font-size: 1rem;
+  font-size: 1.25rem;
```
Comment (helms-charity) [inline aemedge/styles/styles.css:247]: Should be 1.125

**Pull request merged**